packages/desktop_drop/lib/src/channel.dart

```diff
@@ -75,10 +71,8 @@ class DesktopDrop {
         break;
       case "performOperation_linux":
         // gtk notify 'exit' before 'performOperation'.
-        assert(_offset == null);
```

Comment:
The assertion here is to guarantee that:

When the `performOperation` event is received, it is currently in the dropping state, that is, it has received the `entered` event before